setup.cfg

```diff
@@ -0,0 +1,16 @@
+[metadata]
+description-file = README.md
+
+[bdist_wheel]
+universal=1
+
+
+# See the docstring in versioneer.py for instructions. Note that you must
+# re-run 'versioneer.py setup' after changing this section, and commit the
+# resulting files.
+
+[versioneer]
+VCS = git
+style = git-describe
```

Comment:
Note that this produces a non-PEP440-compliant version string for non-release versions:

```
$ python setup.py --version
python3.5/site-packages/setuptools/dist.py:407: UserWarning: The version specified ('0.6.0-22-gc172077') is an invalid version, this may not work as expected with newer versions of setuptools, pip, and PyPI. Please see PEP 440 for more details.
  "details." % self.metadata.version
0.6.0-22-gc172077
```

Personally I prefer `pep440-pre` style (`0.6.0.post0.dev22`)  but  the basic `pep440` style (`0.6.0+22.gc172077`) is closer to `git-describe` style; either one makes setuptools happier.

Reply:
Changed to pep440.